### PR TITLE
[Manual Search] Season manual search

### PR DESCRIPTION
**Pull request by fernandog**
- [ ] Season search makes ep 1 spin....@p0psicles

image

image

### CODE REVIEW

File: sickbeard/manual_search.py [hunk 1]
```diff
@@ -211,20 +211,21 @@ def get_provider_cache_results(indexer, show_all_results=None, perform_search=No
...
-
-        # make a queue item for it and put it on the queue
-        ep_queue_item = search_queue.ForcedSearchQueueItem(ep_obj.show, ep_obj, bool(int(down_cur_quality)), True)  # pylint: disable=maybe-no-member
-
-        sickbeard.searchQueueScheduler.action.add_item(ep_queue_item)
-
-        # give the CPU a break and some time to start the queue
-        time.sleep(cpu_presets[sickbeard.CPU_PRESET])
+        if episode:
+            ep_obj = getEpisode(show, season, episode)
```
Comment (fernandog) [inline sickbeard/manual_search.py]: @p0psicles need to figure it out how to season search. I guess we need to pass multiple episodes

**Comment by p0psicles** [comment]
Do you want to show users all results for all individual episodes in a season? Or just season packs?

**Comment by fernandog** [comment]
Season pack only

File: gui/slick/js/core.js [hunk 2]
```diff
@@ -2549,6 +2549,7 @@ var SICKRAGE = {
                 var season = $('meta[data-last-prov-updates]').attr('data-season');
                 var episode = $('meta[data-last-prov-updates]').attr('data-episode');
                 var data = $('meta[data-last-prov-updates]').data('last-prov-updates');
+                var mode = $(this).attr('data-mode')
```
Comment (p0psicles) [inline gui/slick/js/core.js]: simelicon (;)

**Pull request merged**